jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -1070,6 +1070,14 @@ properties:
                 title: Dark reference file name
                 type: string
                 fits_keyword: R_DARK
+          darkMIRI:
```

Comment:
ok I remved that from the core schema
Jane

On 07/21/2016 06:37 PM, Howard Bushouse wrote:
>
> In jwst/datamodels/schemas/core.schema.yaml 
> <https://github.com/STScI-JWST/jwst/pull/118#discussion_r71815924>:
>
> > @@ -1070,6 +1070,14 @@ properties:
> >                  title: Dark reference file name
> >                  type: string
> >                  fits_keyword: R_DARK
> > +          darkMIRI:
>
> This addition to the core schema is not necessary. It's only the 
> definition of the keyword that will be used to record the name of the 
> dark reference file that was used during calibration. The same keyword 
> "R_DARK" can and should be used for all instruments, hence there's no 
> need for a separate meta data entry.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/STScI-JWST/jwst/pull/118/files/76105243e0379a5934adbb1034a1377a5d1cec05#r71815924>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/ARonWdMQbK2qKZxjbWTs4mmydVHdwayyks5qYB7DgaJpZM4JSExt>.
>